cybergis_compute_client/__init__.py

```diff
@@ -16,4 +16,6 @@
 limitations under the License.
 """
 
+__version__ = "0.2.6"
```

Comment:
Everything here works, but I would prefer to avoid having to hard-code the version here (we currently have it set in the setup.py). I think I found a workaround.

In the UI.py instead of importing the version from the package, you can use the line:

```
import importlib.metadata as importlib_metadata
```

then we should be able to access the version using the variable:

```
importlib_metadata.version("cybergis_compute_client")
```